code/cnn_1D_segm/train_fcn1D.py

```diff
@@ -0,0 +1,364 @@
+
+# coding: utf-8
+
+#!/usr/bin/env python2
```

Comment:
Ideally, that file should work both with Python 2 and Python 3. If not, we will have to convert it later.